components/clarinet-cli/src/frontend/cli.rs

```diff
@@ -1417,6 +1439,14 @@ pub fn main() {
             println!("{} {}", green!("Created file"), file_name.clone());
             println!("Check your shell's documentation for details about using this file to enable completions for clarinet");
         }
+
+        Command::Devnet(subcommand) => match subcommand {
+            Devnet::Package(cmd) => {
+                let manifest = load_manifest_or_exit(cmd.manifest_path);
+                Package::pack(cmd.package_file_name, manifest)
+                    .expect("Could not execute the package command.");
```

Comment:
The `Package::pack` command bubbles up a `Result`, which could have more context. I like the `"Could not execute the package command"` error, but we could add the additional details from the result?

Reply:
Done